client/src/App/components/groups/createEdit/GroupCreateEditSurface.jsx

```diff
@@ -36,6 +36,9 @@ class GroupCreateEditSurface extends React.Component {
         // topics get passed from the form as a single string value, 
         // but must be passed to the server as an array
         group.topics = [group.topics];
+        if (!group.leaders) {
+            group.leaders = [];
```

Comment:
Are you sure we want the default group leaders to be null? I would think we'd want the group leaders to default to the currently logged in user.